src/gsi/hybrid_ensemble_isotropic.F90

```diff
@@ -4017,7 +4051,7 @@ subroutine hybens_localization_setup
    if(verbose .and. mype == 0)print_verbose=.true.
 
    ! Allocate
-   call create_hybens_localization_parameters
+   !call create_hybens_localization_parameters
```

Comment:
This is now called in `glbsoi` during the initialization.  
It can be removed from here, no?

Reply:
yes done